packages/hardhat-core/src/internal/hardhat-network/provider/vm/exit.ts

```diff
@@ -24,6 +24,7 @@ export class Exit {
       case SuccessReason.Stop:
       case SuccessReason.Return:
       case SuccessReason.SelfDestruct:
+      case SuccessReason.EofReturnContract:
```

Comment:
Is this new from the latest EDR and unrelated to Rip-7212?

Reply:
Yes, this is something I should've updated in the previous version bump but didn't.